src/Sarif/Writers/SarifLogger.cs

```diff
@@ -198,7 +198,7 @@ public void Dispose()
                     _issueLogJsonWriter.WriteToolNotifications(_run.ToolNotifications);
                 }
 
-                if (_run != null && 
+                if (_run != null &&
```

Comment:
If you look carefully at the left side, you'll see that a single space at the end of the line has been removed (which is good :-)).